src/applications/mhv/medications/components/shared/ExtraDetails.jsx

```diff
@@ -85,9 +82,8 @@ const ExtraDetails = rx => {
         <div className="no-print">
           <p className="vads-u-margin-y--0">
             We put a hold on this prescription. If you need it now, call your VA
-            pharmacy at <va-telephone contact="3538675309" /> (
-            <va-telephone contact="711" tty />
-            ).
+            pharmacy at
```

Comment:
Looks good, youll just have to remove the "at" since that is inserted in the component. other than that approved 
👍